test/core/rasterization/ellipse.cpp

```diff
@@ -0,0 +1,79 @@
+//
+// Copyright 2021 Prathamesh Tagore <prathameshtagore@gmail.com>
+//
+// Use, modification and distribution are subject to the Boost Software License,
+// Version 1.0. (See accompanying file LICENSE_1_0.txt or copy at
+// http://www.boost.org/LICENSE_1_0.txt)
+//
+#include <boost/core/lightweight_test.hpp>
+#include <boost/gil.hpp>
+#include <array>
+#include <cmath>
+#include <vector>
+
+namespace gil = boost::gil;
+
+// This function utilizes the fact that sum of distances of a point on an ellipse from its foci 
+// is equal to the length of major axis of the ellipse.
+// Parameters b and a represent half of lengths of vertical and horizontal axis respectively.
+void test_rasterizer_follows_equation(std::vector<std::array<std::ptrdiff_t, 2>> trajectory_points, 
+                                                                                    int a, int b)
```

Comment:
Please, don't run with the arguments to the right.
Humans usually read from left to right, so we prefer to align to left, e.g. 

```cpp
void test_rasterizer_follows_equation(
    std::vector<std::array<std::ptrdiff_t, 2>> trajectory_points, 
    int a, int b)
```